components/cd-bullet-list/cd-bullet-list.html.twig

```diff
@@ -1,31 +1,51 @@
+{{ attach_library('common_design/cd-flow') }}
 {{ attach_library('common_design/cd-bullet-list') }}
 
-<ul class="cd-bullet-list">
+<ul class="cd-bullet-list cd-bullet-list--icon [ cd-flow ]">
   <li>
-    <a href="#" rel="tag">
-      <svg class="cd-icon cd-icon--arrow-right">
+    <a href="#">
+      <svg class="cd-icon cd-icon--arrow-right" aria-hidden="true" focusable="false" width="16" height="16">
         <use xlink:href="#cd-icon--arrow-right"></use>
       </svg>
-      Menu item title
+      Sed dapibus lectus sit amet adipiscing egestas mauris est viverra nibh
     </a>
   </li>
 
   <li>
-    <a href="#" rel="tag">
-      <svg class="cd-icon cd-icon--arrow-right">
+    <a href="#">
+      <svg class="cd-icon cd-icon--arrow-right" aria-hidden="true" focusable="false" width="16" height="16">
         <use xlink:href="#cd-icon--arrow-right"></use>
       </svg>
-      Menu item title
+      Sed dapibus lectus sit amet
     </a>
   </li>
 
   <li>
-    <a href="#" rel="tag">
-      <svg class="cd-icon cd-icon--arrow-right">
+    <a href="#">
+      <svg class="cd-icon cd-icon--arrow-right" aria-hidden="true" focusable="false" width="16" height="16">
         <use xlink:href="#cd-icon--arrow-right"></use>
       </svg>
-      Menu item title
+      Adipiscing egestas mauris est viverra nibh
+    </a>
+  </li>
+</ul>
+
+<ul class="cd-bullet-list [ cd-flow ]">
```

Comment:
I'm not sure if this is out of scope for the individual components, but the demo displays these two lists as one. I can't see where one ends and another begins.